@xen-orchestra/lite/src/components/AccountButton.vue

```diff
@@ -1,38 +1,73 @@
 <template>
-  <button class="account-button">
-    <FontAwesomeIcon class="user-icon" :icon="faCircleUser" />
-    <FontAwesomeIcon class="dropdown-icon" :icon="faAngleDown" />
-  </button>
+  <AppMenu placement="bottom-end" shadow>
+    <template #trigger="{ open, isOpen }">
+      <button :class="{ active: isOpen }" class="account-button" @click="open">
+        <UiIcon :icon="faCircleUser" class="user-icon" />
+        <UiIcon :icon="faAngleDown" class="dropdown-icon" />
+      </button>
+    </template>
+    <MenuItem :icon="faGear">Settings</MenuItem>
+    <MenuItem :icon="faMessageExclamation">Send us feedback</MenuItem>
+    <MenuItem
+      :icon="faArrowRightFromBracket"
+      class="menu-item-logout"
+      @click="logout"
+    >
+      Log out
+    </MenuItem>
+  </AppMenu>
 </template>
 
 <script lang="ts" setup>
+import { nextTick } from "vue";
+import { useRouter } from "vue-router";
 import { faAngleDown } from "@fortawesome/pro-light-svg-icons";
+import {
+  faArrowRightFromBracket,
+  faGear,
+  faMessageExclamation,
+} from "@fortawesome/pro-regular-svg-icons";
 import { faCircleUser } from "@fortawesome/pro-solid-svg-icons";
+import AppMenu from "@/components/menu/AppMenu.vue";
+import MenuItem from "@/components/menu/MenuItem.vue";
+import UiIcon from "@/components/ui/UiIcon.vue";
+import { useXenApiStore } from "@/stores/xen-api.store";
+
+const router = useRouter();
+
+const logout = () => {
+  const xenApiStore = useXenApiStore();
+  xenApiStore.disconnect();
+  nextTick(() => router.push({ name: "home" }));
```

Comment:
When we disconnect, we only disconnect from the XAPI and we stay connected to xo-lite, right?

Reply:
Not sure to understand what you mean.

AFAIK, there is no "XO Lite connection", everything happens on XAPI, isn't it?